pandas/tests/io/test_common.py

```diff
@@ -89,6 +107,68 @@ def test_iterator(self):
         tm.assert_frame_equal(first, expected.iloc[[0]])
         tm.assert_frame_equal(concat(it), expected.iloc[1:])
 
+    @pytest.mark.parametrize('reader, module, path', [
```

Comment:
if you are moving these here. then maybe do the same with Path/localpath tests (#16292). can do after of course.

though not sure whether I like better in each module, or centralized.

cc @chris-b1